content/en/gitops.md

```diff
@@ -4,17 +4,35 @@ status: Feedback Appreciated
 category: Concept
 ---
 
+
 ## What it is
+Let's start with DevOps and then move on to GitOps. 
+**DevOps** is a set of methods that automate and integrate processes between development and operations teams. 
+**GitOps** includes the involvement of a version-control system **Git** which increases the security and efficiency by many folds.
+
+GitOps is a set of best practices based on [shared principles](https://opengitops.dev/) that are applied to a workflow that relies on software agents to automate the reconciliation of a declared system state or configuration in a git repository that stores the software's code.
+
+It integrates operations such as Git deployment, management, and monitoring for containerized clusters and softwares.
+
+The Git repository for the project stores the entire state of the system, making the history of system changes visible and editable.
 
-GitOps is a set of best practices based on [shared principles](https://opengitops.dev/), applied to a workflow that depends on software agents that enable automation to reconcile a declared system state or configuration in a git repository.
-These software agents and practices are used to execute a cohesive workflow that leverages a source control system like Git as the “single source of truth” and extends this practice to applications, infrastructure, and operational procedures.
+These software agents and practices are used to execute a cohesive workflow that leverages a source control system like Git as the "single source of truth" and extends this practice to applications, infrastructure, and operational procedures.
 
-## Problem it addresses
 
-Existing processes for infrastructure configuration management can face challenges such as configuration drift, failed deployments, relying on a system's previous state for success, missing documentation, or unknown development history.
-Adopting a GitOps workflow can help alleviate these issues, among several others.
+
+## Problems it addresses
```

Comment:
Problem it addresses should not talk about GitOps at all and should focus on the problem only. What is life without GitOps like?